### DISCUSSION TITLE
Contents of ACAD_TABLE

### DISCUSSION THREAD

**Discussion by JEZHDK** [mozman/ezdxf#512] [Q&A] [answered]
Hi
Thank you for a great python library.

I would like to know if it possible to extract the contents of a ACAD_TABLE? Would be vary useful.

Thanks.

**Comment by mozman** [comment, marked as answer]
No, the ACAD_TABLE entity is not supported yet, and it is also not on the TODO list for 1.0 release to the end of this year (hopefully).

And even then, it will take a long time to support this entity because it is very complex and there is hardly any documentation. I suspect this feature will be available in 2-3 years at the earliest, if at all.

**Reply by JEZHDK** [reply]
Thank you for the reply.

I hope it proves easier then expected.
Thank you for your good work.
